test/Algebra/Graph/Test/AdjacencyMap.hs

```diff
@@ -70,12 +69,3 @@ testAdjacencyMap = do
                                            , (Set.fromList [1,4], Set.fromList [5]  )
                                            , (Set.fromList [3]  , Set.fromList [1,4])
                                            , (Set.fromList [3]  , Set.fromList [5 :: Int])]
-
-    putStrLn "\n============ AdjacencyMap.Internal.GraphKL ============"
```

Comment:
I think this shouldn't be just deleted -- you should add a test module for testing all examples in `Data.Graph.Typed`.